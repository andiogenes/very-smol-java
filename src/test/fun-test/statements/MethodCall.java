class Main {
    int x = 0;

    void foo() {
        println(3);
        x = x + 10;
    }

    void bar() {
        println(2);
        foo();
        x = x + 10;
    }

    void baz() {
        println(1);
        bar();
        x = x + 10;
    }

    void main() {
        baz();
        println(x);
    }
}