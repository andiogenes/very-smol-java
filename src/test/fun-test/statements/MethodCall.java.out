x : INT = 0 : INT
1
2
3
x : INT = 10 : INT
x : INT = 20 : INT
x : INT = 30 : INT
30
